### PR TITLE
Add sorting selector for quotes

### PR DESCRIPTION
**Pull request by Arashfa0301**
Added sorting selector for quotes which now enables users to sort after Imojis 🤌

### CODE REVIEW

**Review by eikhr — CHANGES_REQUESTED** [review]
Good work! I've commented on some small things and a small problem is that it's not possible to go directly to `/quotes/?ordering=-reaction_count` as it will just change to the default quote sorting.

File: app/routes/quotes/components/QuotePage.js [hunk 1]
```diff
@@ -35,6 +39,22 @@ type Props = {
...
 
+type Option = {
+  label: string,
+  value: string,
+};
+
+const filterRegDateOptions: Array<Option> = [
+  {
+    label: 'nyeste',
+    value: '/quotes',
```
[the remark this answers is not in the document]
Comment (Arashfa0301) [reply app/routes/quotes/components/QuotePage.js:50]: Absolutely correct

File: app/routes/quotes/components/QuotePage.js [hunk 2]
```diff
@@ -61,10 +81,36 @@ export default function QuotePage({
...
+
   return (
     <div className={cx(styles.root, styles.quoteContainer)}>
       <Helmet title="Overhørt" />
       {navigation('Overhørt', actionGrant)}
+
+      <div className={styles.select}>
+        <div>Sorter etter:</div>
+        <Select
+          name="form-field-name"
```
[the remark this answers is not in the document]
Comment (Arashfa0301) [reply app/routes/quotes/components/QuotePage.js:104]: Can't forget the name:)

File: app/routes/quotes/components/QuotePage.js [hunk 3]
```diff
@@ -61,10 +81,36 @@ export default function QuotePage({
         ? 'Ingen sitater venter på godkjenning.'
         : 'Fant ingen sitater. Hvis du har sendt inn et sitat venter det trolig på godkjenning.';
   }
+
+  const history = useHistory();
+  const [ordering: Option, setOrdering] = useState(filterRegDateOptions[0]);
```
[the remark this answers is not in the document]
Comment (Arashfa0301) [reply app/routes/quotes/components/QuotePage.js:86]: Learn something new everyday

**Review by eikhr — APPROVED** [review]

**Comment by ivarnakken** [comment]
Please add some illustrations of the result 😄

**Comment by eikhr** [comment]
I just realized you didn't fix the thing about going directly to the sorted page. It always resets to default sorting regardless of the url. Do you want to look into that, too?

**Comment by Arashfa0301** [comment]
> I just realized you didn't fix the thing about going directly to the sorted page. It always resets to default sorting regardless of the url. Do you want to look into that, too?

It is finally fixed now. Thank you for your precious feedback

**Review by eikhr — APPROVED** [review]

**Pull request merged**